Header header
int16 Chg_state
int8 SOC  # State of Charge:percent
int16 Bat_alarm
int16 Bat_cycle
